tests/system/migrations/test_0042_remove_template_fields.py

```diff
@@ -0,0 +1,1320 @@
+                "list_of_speakers_id": 172,
+                "poll_ids": [81],
+            },
+        },
+        # chat messages
+        {
+            "type": "create",
+            "fqid": "chat_message/151",
+            "fields": {
+                "id": 151,
+                "meeting_id": 41,
+                "user_id": 221,
+                "chat_group_id": 191,
+                "content": "message",
+                "created": 1684938947,
+            },
+        },
+        # chat groups
+        {
+            "type": "create",
+            "fqid": "chat_group/191",
+            "fields": {
+                "id": 191,
+                "meeting_id": 41,
+                "chat_message_ids": [151],
+                "name": "chat group",
+            },
+        },
+        # motion workflows
+        {
+            "type": "create",
+            "fqid": "motion_workflow/211",
+            "fields": {
+                "id": 211,
+                "meeting_id": 41,
+                "default_workflow_meeting_id": 41,
+                "default_amendment_workflow_meeting_id": 41,
+                "default_statute_amendment_workflow_meeting_id": 41,
+                "name": "workflow",
+                "sequential_number": 1,
+                "first_state_id": 161,
+                "state_ids": [161],
+            },
+        },
+        {
+            "type": "create",
+            "fqid": "motion_workflow/212",
+            "fields": {
+                "id": 212,
+                "meeting_id": 42,
+                "default_workflow_meeting_id": 42,
+                "default_amendment_workflow_meeting_id": 42,
+                "default_statute_amendment_workflow_meeting_id": 42,
+                "name": "workflow",
+                "sequential_number": 1,
+                "first_state_id": 162,
+                "state_ids": [162],
+            },
+        },
+        # motion states
+        {
+            "type": "create",
+            "fqid": "motion_state/161",
+            "fields": {
+                "id": 161,
+                "meeting_id": 41,
+                "name": "state",
+                "weight": 1,
+                "css_class": "lightblue",
+                "workflow_id": 211,
+                "first_state_of_workflow_id": 211,
+                "motion_ids": [61],
+            },
+        },
+        {
+            "type": "create",
+            "fqid": "motion_state/162",
+            "fields": {
+                "id": 162,
+                "meeting_id": 42,
+                "name": "state",
+                "weight": 1,
+                "css_class": "lightblue",
+                "workflow_id": 212,
+                "first_state_of_workflow_id": 212,
+                "motion_ids": [62],
+            },
+        },
+    )
+    write(
+        {
+            "type": "delete",
+            "fqid": "meeting/43",
+        }
+    )
+    finalize("0042_remove_template_fields")
+
+    assert_model(
```

Comment:
Please assert also the deleted meeting/43

Reply:
done